.setup/INSTALL_SUBMITTY_HELPER_BIN.sh

```diff
@@ -75,3 +76,11 @@ done
 chown -R root:${HWCRON_USER} ${SUBMITTY_INSTALL_DIR}/sbin/autograder
 chmod 750 ${SUBMITTY_INSTALL_DIR}/sbin/autograder
 chmod 550 ${SUBMITTY_INSTALL_DIR}/sbin/autograder/*
+
+if [ "${WORKER}" == 1 ]; then
+    chown -R root:${SUBMITTY_SUPERVISOR} ${SUBMITTY_INSTALL_DIR}/sbin/shipper_utils
+else
+    chown -R root:${HWCRON_USER} ${SUBMITTY_INSTALL_DIR}/sbin/shipper_utils
```

Comment:
_USER here is incorrect.  it should be _GROUP.

But more important we should be consistent.  In some places we're doing root:hwcron_group in other places we're doing hwcron_user:root.  

If hwcron_group only contains hwcron (and I hope this is true), maybe the group is better (contradicting what I said to do earlier...)  

And let's make sure hwcron only has read in most of these instances, no write (unless of course it needs write)